_includes/tailwind-styles.html

```diff
@@ -251,6 +251,102 @@
             @apply left-full ml-[2px] top-1/2 -translate-y-1/2;
         }
 
+        /*
+        Define Tables
+        */
+
+        .htk-table {
+            @apply max-w-full text-sm text-gray-700 text-center;
+        }
+
+        .htk-table-head {
+            @apply border-b-2 border-neutral-200 tracking-wider text-neutral-500;
+        }
+
+        .hkt-table-row {
+            @apply border-b border-neutral-200;
+        }
+
+        .hkt-table-row:hover {
+            @apply bg-neutral-100;
+        }
+
+        .hkt-table-row-name {
+            @apply tracking-wider text-neutral-500;
+        }
+
+        .hkt-table-footer {
+            @apply border-t border-neutral-200;
+        }
+
+        .hkt-pagination {
+            @apply flex justify-between items-center;
+        }
+        /*
+        Define Text Inputs
+        */
+
+        .hkt-text-input {
```

Comment:
For `text-input` and `text-with-icon-input`:

I suggest just setting both to `.htk-text-input` then tacking on another class for saying whether it has icon. Since I think the style will be the same on these icons no matter what type of input it is, I think it can be a shared class - ie. `.htk-input-with-icon` or something like that.

Reply:
Just an added thought:

A lot of libraries give the ability to specify a prepended or appended icon. I'm not sure we'd need to be this specific, but it might be worth looping Jeff or David in when we discuss. But if we go that route it can look like `.htk-input-with-icon` would just add the icon at the end of the input and `.htk-input-with-icon-pre` would specify that the icon was prepended.